editing_and_imputation/imputation/ratio_of_means/methodological_specification.md

```diff
@@ -410,4 +490,62 @@ Please see the image below for further information.
 
 ![imputation_types1](https://user-images.githubusercontent.com/87982871/167370091-bd18e5bb-fef5-4d46-9b1e-a452040d9e16.png)
```

Comment:
If it's not too much work, I'd include the MC and FIMC scenarios in the image.

Reply:
Added in a new image with the 8 scenarios listed in the specification.